MATCHERS.md

```diff
@@ -70,6 +71,36 @@ state: s.enum({values: ['NSW', 'VIC'], name:'state'})
 state: s.enum({values: ['NSW', 'VIC'], name:'state', verbose: true})
 ```
 
+## exactObject
+
+Match an exact object.
+This matcher will not error if any optional properties are left out.
+This matcher will error if any extra properties passed in.
+
+Each property must have a corresponding matcher, and they will be called recursively.
+
+```js
+// match exact object
+person: 'exactObject'
+person: s.exactObject()
```

Comment:
that is a possibility, not sure how much of the code that would affect
I think a lot of it is expect a single argument

@rprieto thoughts?